config/amip_configs/amip.yml

```diff
@@ -0,0 +1,23 @@
+FLOAT_TYPE: "Float32"
+albedo_model: "CouplerAlbedo"
+anim: false
+atmos_config_file: "config/longrun_configs/amip_target_diagedmf.yml"
```

Comment:
Would it be better to just have all the configs here, instead of inheriting from an atmos job?